news/1520.bugfix

```diff
@@ -0,0 +1 @@
+In the Tutorials doc, updated the URL to download the ``notebook.py`` script from GitHub.
```

Comment:
Because this pr is so minor, I think you can change the type from *bugfix* to *misc* so that it does't pollute the bug fix section of the changelog.

Reply:
Done!